client/index.jsx

```diff
@@ -25,6 +26,8 @@ const App = () => (
 const HotApp = hot(App);
 
 render(
-  <HotApp />,
+  <Suspense fallback={(<div>Loading translations</div>)}>
```

Comment:
Let's use the [`<Loader />`](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/App/components/loader.jsx) here :-)